tools/update-translator-sources

```diff
@@ -34,9 +34,20 @@ update_template_file() {
   sed -i -e 's/charset=CHARSET/charset=UTF-8/g' "${pot_temp_file}"
 
   mkdir -p po
+
+
   msgmerge --backup=none --update -- "${pot_file}" "${pot_temp_file}"
   sed -i -e '/^.POT-Creation-Date:/d' "${pot_file}"
 
+	if [ $(cat $pot_file | grep -c "Code generated by") -eq 0 ]; then
+		sed -i '1s;^;# Code generated by tools/update-translator-sources. DO NOT EDIT.\n;' "$pot_file"
+		sed -i '2s;^;# source: benchmark/benchmark-translation.cpp\n;' "$pot_file"
+		sed -i '3s;^;# source: src/diagnostic-formatter.cpp\n;' "$pot_file"
+		sed -i '4s;^;# source: src/quick-lint-js/diagnostic-types.h\n;' "$pot_file"
+		sed -i '5s;^;# source: test/test-diagnostic-formatter.cpp\n;' "$pot_file"
+		sed -i '6s;^;# source: test/test-vim-qflist-json-diag-reporter.cpp\n\n;' "$pot_file"
```

Comment:
Avoid duplicating the list of files from line 29.

I think we can just write `source: (various)` here. No need to be thorough.